content/applications/productivity/whatsapp.rst

```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+window will appear asking which app this token should be generated for. Select the :guilabel:`App`
+that this token is for. Then determine the expiration date of either :guilabel:`60 days` or
+:guilabel:`Never`.
+
+Finally when Meta asks which permissions should the system user allow, add all of the following
+permissions:
+
+- instagram_manage_comments
+- instagram_basic
+- business_management
+- read_insights
+- pages_show_list
+- ads_read
+- instagram_manage_insights
+- catalog_management
+- pages_messaging
+- page_events
+- publish_video
+- leads_retrieval
+- pages_read_engagement
+- pages_manage_metadata
+- pages_manage_ads
+- pages_manage_cta
+- whatsapp_business_messaging
+- ads_management
+- pages_read_user_content
+- pages_manage_posts
+- whatsapp_business_management
+- instagram_manage_messages
+- pages_manage_instant_articles
+- read_page_mailboxes
+- instagram_content_publish
+- pages_manage_engagement
+
+Click :guilabel:`Generate token`. Copy the token that populates on the following screen.
+
+Update the :guilabel:`Access Token` in the WhatsApp business account in Odoo. :ref:`webhooks`.
+
+Go live with the Meta app
+-------------------------
+
+Finally, to complete the setup, the Meta app must be set to :guilabel:`Live` in the Meta developer
+console to launch the app. Navigate to `<https://developers.facebook.com/apps>`_ and click on the
+app that is being configured. In the top menu, toggle the :guilabel:`App Mode` field from
+:guilabel:`Development` to :guilabel:`Live`.
+
+.. note::
+   If the app status  is not set to *live*, then the database will only be able to contact the test
+   numbers specified in the developer console.
+
+.. warning::
+   A privacy policy URL must be set in order for the app to be set to live. Go to the Meta developer
+   console, `<https://developers.facebook.com/apps>`_ and select the app that Odoo is being
+   configured in. Then in the left hand menu, under :guilabel:`App settings`, select
+   :guilabel:`Basic`. Enter the privacy policy under :guilabel:`Privacy Policy URL`. Click
+   :guilabel:`Save changes` to apply the privacy policy to the app.
+
+A email is sent to the administrator once the app has gone live in the Meta developer console.
+
+.. _whatsapp/watemplates:
+
+WhatsApp templates
+==================
+
+WhatsApp templates are saved messages that are used repeatedly to send messages from the database.
+They allow users to send quality communications, without having to compose the same text repeatedly.
+
+Creating different templates that are tailored to specific situations lets users choose the right
+message for the right audience. This increases the quality of the message and the overall
+engagement rate with the customer.
+
+WhatsApp templates can be created on both the Odoo and Meta consoles. The following process will
+overview the process for creating templates in Odoo.
+
+.. important::
+   WhatsApp has an approval process that must be completed before the template can be used.
```

Comment:
May be worth linking to the doc explaining the approval process (or to the section below where you explain it) in case a reader sees this warning but has additional questions.

Reply:
Added anchor